asreview/webapp/task_manager/task_manager.py

```diff
@@ -0,0 +1,258 @@
+import argparse
+import json
+import logging
+import socket
+import threading
+from collections import deque
+
+from sqlalchemy import create_engine
+from sqlalchemy import text
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy.orm import sessionmaker
+
+from asreview.webapp import asreview_path
+from asreview.webapp.task_manager.models import Base
+from asreview.webapp.task_manager.models import ProjectQueueModel
+from asreview.webapp.task_manager.task_wrapper import RunModelProcess
+from asreview.webapp.tasks import run_task
+
+
+class TaskManager:
+    def __init__(self, max_workers=1, host="localhost", port=5555):
```

Comment:
```suggestion
    def __init__(self, max_workers=1, host="localhost", port=5555):
```

Shall we set the max_workers here also at 2? Of the other one at 1? It's nice to align. 

Port 5555 sounds like a port that might be popular for some other applications as well. Maybe use a less popular port as default?

Reply:
Fully agree, port 5101 and 2 workers